lib/bundle/brew_installer.rb

```diff
@@ -31,6 +32,10 @@ def install_change_state!
       end
     end
 
+    def service_needs_started?
```

Comment:
@voanhduy1512 Sorry: absolutely last thing: can you renamed this to `start_service?` Thanks!